po/de.po

```diff
@@ -1833,14 +1831,12 @@ msgid "Error: unsupported URL: %s"
 msgstr "Fehler: Nicht unterstützte URL: %s"
 
 #: src/searchresultslistformaction.cpp:18
-#, fuzzy
 msgid "Prev search results"
-msgstr "Suchergebnis - „%s“"
+msgstr "Vorheriges Suchergebnis - „%s“"
 
 #: src/searchresultslistformaction.cpp:63
-#, fuzzy
 msgid "Already in first search result."
```

Comment:
Maybe something for the future: I noticed that "search results" is always plural except here. Might be fine, though, because it sounds a wee bit nicer to my ears. Maybe some native speaker can shed some light.

In the German translations I now went with singular everywhere (changed line 270 from plural to singular), since it's shorter and you can basically use both.

Reply:
I actually want to replace this message entirely, but haven't found the words to say it any better =\ Good call on the plural through!

Reply:
I cannot think of a better phrase. Maybe not showing anything and just doing nothing. But I hardly ever use the search myself, so not sure if this is a user-friendly option in the new search stack behavior.